db/migrate/20150221153705_create_submissions.rb

```diff
@@ -1,11 +1,13 @@
 class CreateSubmissions < ActiveRecord::Migration
   def change
     create_table :submissions do |t|
-      t.timestamps null: false
       t.references :user, null: false
       t.string :step, null: false
       t.json :content, null: false, default: {}
-      t.boolean :finalized, null: false, default: false
+      t.boolean :finalized, null: false, default: false, index: true
```

Comment:
I do not see any problem with growing number of migrations (I see problem with old migrations which fail but that is quite different thing). However changing existing migration forces other devs to partially recreate their development db... Just my thoughts, I am not saying you should do anything in this case.